tstest/integration/vms/vms_test.go

```diff
@@ -52,6 +54,17 @@ var (
 	}()
 )
 
+func TestMain(m *testing.M) {
+	flag.Parse()
+	err, cleanup := integration.BuildTestBinaries()
```

Comment:
I'd rather `sync.Once` it so we don't need to build binaries until/if they're first needed.

Reply:
Done